trie/trienode/proof.go

```diff
@@ -102,14 +102,26 @@ func (db *ProofSet) DataSize() int {
 	return db.dataSize
 }
 
-// List converts the node set to a ProofList
-func (db *ProofSet) List() ProofList {
+// ByteList converts the node set to a [][]byte
+func (db *ProofSet) ByteList() [][]byte {
 	db.lock.RLock()
 	defer db.lock.RUnlock()
 
-	var values ProofList
-	for _, key := range db.order {
-		values = append(values, db.nodes[key])
+	var values = make([][]byte, len(db.order))
+	for i, key := range db.order {
+		values[i] = db.nodes[key]
+	}
+	return values
+}
+
+// ProofList converts the node set to a ProofList
+func (db *ProofSet) ProofList() ProofList {
```

Comment:
It seems to me that only `ByteList()` is ever used. Can we just drop this method?

Reply:
Yeah, it has never been used. delete it now, will add it later if needed.